1.10/installing/custom/configuration/configuration-parameters.md

```diff
@@ -501,3 +501,29 @@ Indicates whether to enable the DC/OS proxy.
 For more information, see the [examples](/docs/1.10/installing/custom/configuration/examples/#http-proxy).
 
 **Important:** You should also configure an HTTP proxy for [Docker](https://docs.docker.com/engine/admin/systemd/#/http-proxy).
+
+### <a id="zk-superuser"></a> zk_super_credentials (Enterprise DC/OS Only)
+
+In `strict` and `permissive` clusters the information stored in ZooKeeper is protected by ACLs so that a malicious user can't connect to ZooKeeper quorum and directly modify service metadata. ACLs specify sets of IDs and permissions that are associated with those IDs. ZooKeeper supports pluggable authentication schemes and has a few built in schemes: `world`, `auth`, `digest`, `host`, `ip`. 
```

Comment:
> In `strict` and `permissive` clusters

From the rest of the documentation, we say things like:

"On DC/OS clusters in strict and permissive mode...".

Reply:
The first time we use ACL in a section let's use it like we do in some of our other documentation:

> access control lists (ACLs)

Reply:
> protected by ACLs

ACLs don't protect anything.
Instead, perhaps "protected using ACLs"?

Reply:
> can't

Let's use "cannot" and ideally this would be in a style guide. For now it is my personal, internal style guide that I'm happy to discuss.

Reply:
> ZooKeeper quorum 

I believe that this should be "the ZooKeeper Quorum".